docs/guide/CSS.md

```diff
@@ -468,3 +468,92 @@ For instance, if we have a widget with a (CSS) class called `dialog`, we could r
 
 Note that `initial` will set the value back to the value defined in any [default css](./widgets.md#default-css).
 If you use `initial` within default css, it will treat the rule as completely unstyled.
+
+
+## Nesting CSS
+
+!!! tip "Added in version 0.47.0"
+
+CSS rule sets may be *nested*, i.e. they can contain other rule sets.
+When a rule set occurs within an existing rule set, it inherits the selector from the enclosing rule set.
+
+Let's put this in to practical terms.
+The following example will display two boxes containing the text "Yes" and "No" respectively.
+These could eventually form the basis for buttons, but for this demonstration we are only interested in the CSS.
+
+=== "nesting01.tcss (no nesting)"
+
+    ```sass
+    --8<-- "docs/examples/guide/css/nesting01.tcss"
+    ```
+
+=== "nesting01.py"
+
+    ```python
+    --8<-- "docs/examples/guide/css/nesting01.py"
+    ```
+
+=== "Output"
+
+    ```{.textual path="docs/examples/guide/css/nesting01.py"}
+    ```
+
+The CSS is quite straight forward; there is a rule for the container, all buttons, and a rule each for the yes and no buttons.
```

Comment:
```suggestion
The CSS is quite straightforward; there is a rule for the container, all buttons, and a rule each for the yes and no buttons.
```

Reply:
I'd go even further.
As a non-native English speaker, from the sentence alone I need to read it 3 or 4 times to be sure of how many rules in total there are.
Something slightly more verbose may be exponentially clearer:

```suggestion
The CSS is quite straightforward; there is one rule for the container, one for all buttons, and one rule for each of the buttons.
```